Error HH404: File @openzeppelin/contracts/utils/ReentrancyGuard.sol, imported from contracts/EnhancedTheTruth.sol, not found.

HardhatError: HH404: File @openzeppelin/contracts/utils/ReentrancyGuard.sol, imported from contracts/EnhancedTheTruth.sol, not found.
    at Resolver.resolveImport (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/resolver.ts:221:17)
    at /home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/dependencyGraph.ts:188:28
    at async Promise.all (index 6)
    at DependencyGraph._addDependenciesFrom (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/dependencyGraph.ts:186:5)
    at async Promise.all (index 0)
    at Function.createFromResolvedFiles (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/dependencyGraph.ts:15:5)
    at Environment._runTaskDefinition (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/core/runtime-environment.ts:351:14)
    at Environment.run (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/core/runtime-environment.ts:184:14)
    at SimpleTaskDefinition.action (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/builtin-tasks/compile.ts:1398:58)
    at Environment._runTaskDefinition (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/core/runtime-environment.ts:351:14)

    Caused by: HardhatError: HH402: File @openzeppelin/contracts/utils/ReentrancyGuard.sol doesn't exist.
        at Resolver._validateSourceNameExistenceAndCasing (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/resolver.ts:544:15)
        at Resolver._resolveLibrarySourceName (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/resolver.ts:354:7)
        at Resolver.resolveSourceName (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/resolver.ts:103:22)
        at Resolver.resolveImport (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/resolver.ts:194:24)
        at /home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/dependencyGraph.ts:188:28
        at async Promise.all (index 6)
        at DependencyGraph._addDependenciesFrom (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/dependencyGraph.ts:186:5)
        at async Promise.all (index 0)
        at Function.createFromResolvedFiles (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/dependencyGraph.ts:15:5)
        at Environment._runTaskDefinition (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/core/runtime-environment.ts:351:14)
    
        Caused by: HardhatError: HH1005: Solidity source file @openzeppelin/contracts/utils/ReentrancyGuard.sol not found
            at getSourceNameTrueCase (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/utils/source-names.ts:219:13)
            at validateSourceNameExistenceAndCasing (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/utils/source-names.ts:102:30)
            at Resolver._validateSourceNameExistenceAndCasing (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/resolver.ts:536:7)
            at Resolver._resolveLibrarySourceName (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/resolver.ts:354:7)
            at Resolver.resolveSourceName (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/resolver.ts:103:22)
            at Resolver.resolveImport (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/resolver.ts:194:24)
            at /home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/dependencyGraph.ts:188:28
            at async Promise.all (index 6)
            at DependencyGraph._addDependenciesFrom (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/dependencyGraph.ts:186:5)
            at async Promise.all (index 0)
        
            Caused by: FileNotFoundError: File /home/runner/work/The_Truth/The_Truth/node_modules/@openzeppelin/contracts/utils/ReentrancyGuard.sol not found
                at getFileTrueCase (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/util/fs-utils.ts:152:9)
                at getFileTrueCase (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/util/fs-utils.ts:143:9)
                at getFileTrueCase (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/util/fs-utils.ts:143:9)
                at getFileTrueCase (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/util/fs-utils.ts:143:9)
                at getSourceNameTrueCase (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/utils/source-names.ts:215:22)
                at validateSourceNameExistenceAndCasing (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/utils/source-names.ts:102:30)
                at Resolver._validateSourceNameExistenceAndCasing (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/resolver.ts:536:7)
                at Resolver._resolveLibrarySourceName (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/resolver.ts:354:7)
                at Resolver.resolveSourceName (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/resolver.ts:103:22)
                at Resolver.resolveImport (/home/runner/work/The_Truth/The_Truth/node_modules/hardhat/src/internal/solidity/resolver.ts:194:24)
Error: Process completed with exit code 1.